lib/rucio/common/stomp_utils.py

```diff
@@ -202,13 +202,13 @@ def _parse_config(self, config_section: str) -> StompConfig:
         port = config_get_int(config_section, 'port')
         vhost = config_get(config_section, 'broker_virtual_host', raise_exception=False)
         destination = config_get(config_section, "destination")
-        key_file = config_get(config_section, 'ssl_key_file', default=None, raise_exception=False)
-        cert_file = config_get(config_section, 'ssl_cert_file', default=None, raise_exception=False)
-        username = config_get(config_section, 'username', default=None, raise_exception=False)
-        password = config_get(config_section, 'password', default=None, raise_exception=False)
-        nonssl_port = config_get_int(config_section, 'nonssl_port', default=0, raise_exception=False)
-        timeout = config_get_float(config_section, 'timeout', default=None, raise_exception=False)
-        heartbeats = tuple(config_get_list(config_section, 'heartbeats', default=[0., 0.], raise_exception=False))
+        key_file = config_get(config_section, 'ssl_key_file', raise_exception=False)
+        cert_file = config_get(config_section, 'ssl_cert_file', raise_exception=False)
+        username = config_get(config_section, 'username', raise_exception=False)
+        password = config_get(config_section, 'password', raise_exception=False)
+        nonssl_port = config_get_int(config_section, 'nonssl_port', default=0)
+        timeout = config_get_float(config_section, 'timeout', raise_exception=False)
+        heartbeats = tuple(config_get_list(config_section, 'heartbeats', default=[0., 0.])
```

Comment:
this should fix the syntax errors mentioned in the CI

```suggestion
        heartbeats = tuple(config_get_list(config_section, 'heartbeats', default=[0., 0.]))
```